Automattic/jetpack
# IDC: non-admin screen

#### Changes proposed in this Pull Request:
- create the non-admin IDC screen
- adjust Connection Manager UI package to be available for non-admins so they could see the IDC screen there

#### Jetpack product discussion
p9dueE-3pz-p2#comment-6479

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Rebuild the Connection Manager package: `jetpack build packages/connection-ui`.
2. Connect Jetpack, go to Jetpack Debug and activate the IDC Simulator.
3. Go to "Jetpack Debug -> IDC Simulator", enable "IDC Simulation" and enter a "Spoof URL".
4. Edit a post and save to make the site sync. Wait a few seconds.
5. Go to "Tools -> Connection Manager", confirm that the RNA IDC screen appears.
6. Log in as non-admin, go to "Tools -> Connection Manager". Confirm that the IDC screen looks like the attached screenshot, and the "Learn more..." link works.
<img width="837" alt="Screen Shot 2021-12-08 at 9 57 21 AM" src="https://user-images.githubusercontent.com/1341249/145230404-22bed7b7-2bbe-4ad2-963f-d2eaf1a0a0d3.png">

##### Storybook
7. Go to `projects/js-packages/storybook`, run `pnpm install` and `pnpm run storybook:dev`.
8. When the Storybook loads, navigate to "Identity Crisis -> Admin Screen", find the story "Non Admin". Confirm that it looks like the screenshot.
